layouts/index.html

```diff
@@ -0,0 +1,57 @@
+{{ define "header" }}
+<div class="jumbotron-custom">
+  <div class="logo-ctn">
+    <img
+      class="banner-logo"
+      src="./images/bannerlogo.png"
+      alt="Entrepreneurial Open Source"
+    />
+    <a
+      class="guest-btn"
+      target="_blank"
+      href="https://docs.google.com/forms/d/e/1FAIpQLSeM9Wh9KsvppT0IFvq5nONIFFXZOqgbNP_Pl__X-saDq4qYcQ/viewform?usp=sf_link"
+    >
+      <img
+        class="dialog-box"
+        src="./images/dialogBox.svg"
+        alt="Be a guest button"
+      />
+      <span>Be A Guest</span>
+    </a>
+  </div>
+  <div class="header-ctn">
+    <h1>Entrepreneurs at the</h1>
+    <h1>intersection of</h1>
+    <h1>open source and business</h1>
+  </div>
+</div>
+
+{{ end }} {{ define "main" }}
+
+<div
+  style="
+    background-color: #f7941d;
+    height: 40px;
+    padding: 0 86.5px;
+    font-weight: 600;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: white;
+    margin: 0 -15px;
+  "
+>
+  <div class="container">
+    <span class="margin-0">
+      Listen: <span>Spotify</span> <span>Apple</span> <span>Google</span>
```

Comment:
we should probably consider making this an inline list rather than 3 sequential spans. It would read better for screen readers and bots. An example would be `<ul class="list-inline"><li>...</li></ul>`